### ISSUE TITLE
How to comment on a published article?

### ISSUE THREAD

**Issue by floe** [journalovi/journalovi.github.io#29]
AFAICT this is not something that OJS can support out of the box. Ideas welcome...

**Comment by mjskay** [comment]
Ahhh interesting. For the experimental track, I think we can have github issues, and maybe even think about one of those systems that allow comments on a webpage to be backed by github issues...

For the normal track, we could also try one of those things (like https://hypothes.is ?). Or maybe people should submit such things as commentaries?

**Comment by mjskay** [comment]
But the question with commentaries might be how to review those...

**Comment by floe** [comment]
And the barrier is probably much higher to submit an explicit commentary article, than just writing a few words. I had forgotten about hypothes.is, I think this could be a good idea to integrate.

**Comment by floe** [comment]
Had a look and it seems pretty straightforward, if we can somehow nudge Quarto to shunt a `<script>` tag in there. https://web.hypothes.is/help/embedding-hypothesis-in-websites-and-platforms/

**Comment by floe** [comment]
There's even an OJS plugin: https://github.com/asmecher/hypothesis (or we integrate the script directly, as per https://web.hypothes.is/help/embedding-hypothesis-in-websites-and-platforms/ ).

**Comment by mjskay** [comment]
yeah, though then we also have to make sure we're moderating comments if we use something like hypothesis, right? maybe we should test this out or something... could test it out on the mission statement PDF if we wanted?

**Comment by mjskay** [comment]
Let's not announce it yet, since it might be good to test it out and decide if it's the style of commenting we want?
